core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/GaussianOnlineClassifier.kt

```diff
@@ -100,7 +119,12 @@ class GaussianOnlineClassifier : AIModel {
     }
 
     override fun estimateAccuracy(endpoint: Endpoint): Double {
-        TODO("Not yet implemented")
+        // Strict accuracy
+        //TODO this accuracy can be dynamic for example based on the last 100 requests
+        if (this.tot == 0) {
+            return 0.0
+        }
+        return this.cp.toDouble() / this.tot
```

Comment:
what is the difference of this with `getAccuracy`? shouldn't this method just call instead of duplicating the logic?